### PR TITLE
remove permissions before remove ws service acc

### PR DESCRIPTION
**Pull request by romanbracinik**
Pridavam mazanie permissions pri mazani WS service acc, testy este failnu na suite ohladne sharovania to fixujem tu.
https://github.com/keboola/php-storage-driver-bigquery/pull/51

### CODE REVIEW

File: tests/functional/UseCase/Workspace/CreateDropWorkspaceTest.php [hunk 1]
```diff
@@ -233,6 +234,25 @@ public function testCreateDropWorkspace(): void
...
+        $serviceAccRoles = [];
+        foreach ($actualPolicy as $policy) {
+            $membersString = json_encode($policy->getMembers());
+            assert(is_string($membersString));
+            if (stripos($membersString, 'deleted:serviceAccount:' . $wsServiceAccEmail) !== false) {
+                $serviceAccRoles[] = $policy->getRole();
+            }
+        }
+
+        $this->assertEmpty($serviceAccRoles);
```
Comment (romanbracinik) [inline tests/functional/UseCase/Workspace/CreateDropWorkspaceTest.php:252]: Ked si napr cherry picknem commit s testami do mastru tak to failne pretoze tam budu permissions s email ws service acc zacinajuceho na `deleted:serviceAccount:`

**Comment by romanbracinik** [comment]
Screenshot 2023-08-08 at 19 22 33
Screenshot 2023-08-08 at 19 24 37

V dalsom PR skusim pridat tie retries

**Comment by romanbracinik** [comment]
Tu som pridal retries tak to preslo https://github.com/keboola/php-storage-driver-bigquery/pull/53

File: tests/functional/UseCase/Workspace/CreateDropWorkspaceTest.php [hunk 2]
```diff
@@ -233,6 +234,25 @@ public function testCreateDropWorkspace(): void
         );
 
         $this->assertNull($datasets->getIterator()->current());
+
+        $cloudResourceManager = $this->clientManager->getCloudResourceManager($this->projectCredentials);
+        $actualPolicy = $cloudResourceManager->projects->getIamPolicy(
+            'projects/' . $projectId,
+            (new Google_Service_CloudResourceManager_GetIamPolicyRequest()),
```
Comment (zajca) [inline tests/functional/UseCase/Workspace/CreateDropWorkspaceTest.php]: Tady by to mělo byt imho GetIamPolicyRequest ne ten PSR-0 namespace, ale jinak je to asi cajk.

**Review by zajca — APPROVED** [review]
Zajímavé

File: tests/functional/UseCase/Workspace/CreateDropWorkspaceTest.php (hunk 2 above)
Comment (romanbracinik) [reply tests/functional/UseCase/Workspace/CreateDropWorkspaceTest.php]: Jo dik upravil som to este

**Review by zajca — APPROVED** [review]

**Pull request merged**